cli/cmd/sync_v3.go

```diff
@@ -361,11 +362,30 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 		}
 	}
 
-	totals := sourceClient.Metrics()
+	syncSummaries := make([]syncSummary, len(destinationsClients))
 	for i := range destinationsClients {
 		m := destinationsClients[i].Metrics()
 		totals.Warnings += m.Warnings
 		totals.Errors += m.Errors
+		syncSummaries[i] = syncSummary{
```

Comment:
Another question, what's the reason we're flattening the data? We could have a top level array of sources, then in each source point to an array of destinations with the summary

Reply:
We've decided to release this as preview and keep the data flat for now until further feedback